operator/docs/releases.md

```diff
@@ -24,6 +24,6 @@ We use semantic versioning convention (semver) for stable releases, release bran
    1. Add relatedImages to the CSV with the SHA digests.
    1. Review changes, commmit, and submit the PR for review
 1. Once the release PR is ready and merged, add it to the index image and push to quay.io
-   1. `CATALOG_BASE_IMG=quay.io/konveyor/forklift-operator-index:release-v2.2.0 VERSION=2.3.0 make catalog-build bundle-push`
-   1. `CATALOG_BASE_IMG=quay.io/konveyor/forklift-operator-index:release-v2.3.0 TAG=latest make catalog-build catalog-push`
+   1. `CATALOG_BASE_IMG=quay.io/kubev2v/forklift-operator-index:release-v2.2.0 VERSION=2.3.0 make catalog-build bundle-push`
```

Comment:
This may be a typo and is unrelated to the PR, but why does it say `v2.2.0` here? Shouldn't it be `v2.3.0`?

Reply:
yeah, and those tags don't exist in quay.io/kubev2v - dropped those changes for now